src/mixins/nr-affiliation-mixin.ts

```diff
@@ -151,8 +152,21 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
           }
         }
       }
+    } else if (nr.request_action_cd === 'AML') {
```

Comment:
This looks like it won't work as expected.

Line 143 says "if not a firm then create an IA". But amalgamations are not firms, so an IA would be created, which is incorrect.